src/basic_recipes/error_and_rangebars.jl

```diff
@@ -187,19 +187,18 @@ function _plot_bars!(plot, linesegpairs, is_in_y_direction)
 
     scene = parent_scene(plot)
 
-    whiskers = lift(plot, linesegpairs, scene.camera.projectionview,
-        scene.camera.pixel_space, whiskerwidth) do pairs, _, _, whiskerwidth
```

Comment:
With `camera.pixel_space` this sometimes fails to update correctly when zooming or resizing. `scene.px_area` works fine.